.deploystack/test

```diff
@@ -78,10 +78,33 @@ function evalTest() {
     printf "$expected is ok\n"
 }
 
+function generateProject(){
+    local __resultvar=$1
+    local __STACKSUFFIX=$2
+    local __RANDOMSUFFIX=$(
+        LC_ALL=C tr -dc 'a-z0-9' </dev/urandom | head -c 8
+        echo
+    )
+    local __DATELABEL=$(date +%F)
+    local VALUE=ds-test-$__STACKSUFFIX-$__RANDOMSUFFIX
+    local __BA=$(gcloud beta billing accounts list --format="value(ACCOUNT_ID)" --limit=1 | xargs)
+   
+    gcloud projects create $VALUE --labels="deploystack-disposable-test-project=$__DATELABEL" --folder="155265971980"
+    gcloud beta billing projects link $VALUE --billing-account=$__BA
+    eval $__resultvar="'$VALUE'"
+}
```

Comment:
**Praise:** Thanks for modularizing this code into a separate function. It made it easier for me to review the code.